administrator/components/com_content/tmpl/articles/default_batch_body.php

```diff
@@ -42,7 +45,7 @@
 				<?php echo LayoutHelper::render('joomla.html.batch.tag', []); ?>
 			</div>
 		</div>
-		<?php if ($user->authorise('core.admin', 'com_content')) : ?>
```

Comment:
I think that's the wrong approach, you should ask the service if the workflow is enabled for this view. Also as example for other developer